starter-code/scripts/articleView.js

```diff
@@ -79,10 +102,18 @@ articleView.setTeasers = function() {
   //       process any .read-on clicks that happen within child nodes.
 
   // STRETCH GOAl!: change the 'Read On' link to 'Show Less'
-
+  $('.read-on').on('click', function() {
```

Comment:
Another way to do this is 
`  $('#articles').on('click', '.read-on', function(event) {`